cmd/minikube/cmd/stop.go

```diff
@@ -36,7 +36,12 @@ itself, leaving all files intact. The cluster can be started again with the "sta
 		fmt.Println("Stopping local OpenShift cluster...")
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
-
+		if constants.Registrator.CompatibleWithDistribution(){
```

Comment:
Currently this is something which is not working during the stop process and giving a `panic`. I need to find how it is possible to use same registrator object for unregister for stop/delete.

```
Stopping local OpenShift cluster...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x64a14]

goroutine 1 [running]:
panic(0xc51ac0, 0xc4200100a0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/minishift/minishift/cmd/minikube/cmd.glob..func13(0x14eb680, 0x1515eb0, 0x0, 0x0)
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/cmd/minikube/cmd/stop.go:39 +0x334
github.com/minishift/minishift/vendor/github.com/spf13/cobra.(*Command).execute(0x14eb680, 0x1515eb0, 0x0, 0x0, 0x14eb680, 0x1515eb0)
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/vendor/github.com/spf13/cobra/command.go:603 +0x439
github.com/minishift/minishift/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x14ea9c0, 0x0, 0x0, 0x14b8c20)
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/vendor/github.com/spf13/cobra/command.go:689 +0x367
github.com/minishift/minishift/vendor/github.com/spf13/cobra.(*Command).Execute(0x14ea9c0, 0x0, 0x2a)
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/vendor/github.com/spf13/cobra/command.go:648 +0x2b
github.com/minishift/minishift/cmd/minikube/cmd.Execute()
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/cmd/minikube/cmd/root.go:101 +0x31
main.main()
	/Users/prkumar/work/github/go_practice/src/github.com/minishift/minishift/cmd/minikube/main.go:26 +0x19
```